src/aui-form-builder/js/aui-form-builder-field-list.js

```diff
@@ -172,6 +172,16 @@ A.FormBuilderFieldList  = A.Base.create('form-builder-field-list', A.Base, [], {
             container.append(field.get('content'));
         });
 
+        setTimeout(function() {
```

Comment:
I am not satisfied by this, but I can't see a clearer way to do it with this organization we have for the form builder. Any ideas?